Contribution guide update (#3643)
timescale/docs: README.md

To maintain consistency, please follow these general rules.

*   Maintain text editor width for paragraphs at 80 characters. We ask you to do
    this to assist in reviewing documentation changes. When text is very wide, it
    is difficult to visually see where text has changed within a paragraph and keeping
    a narrow width on text assists in making PRs easier to review. **Most editors such
    as Visual Studio Code have settings to do this visually.**
*   Most links should be reference-style links where the link address is at the
    bottom of the page. The two exceptions are:
    *   Links within highlight blocks (Note, Important, or Warning). These must be inline links for now
    *   Links to anchors on the same page as the link itself.

> *   All functions, commands and standalone function arguments (ex. `SELECT`,

Reviewer: ```suggestion
*   Set all functions, commands and standalone function arguments such as `SELECT`,
```

Author: This will be removed from Readme, but I will do it in a separate PR

>     `time_bucket`) should be set as inline code within backticks ("\`command\`").

Reviewer: ```suggestion
    and `time_bucket` as inline code within backticks ("\`command\`").
```

Author: This too
